src/test/java/Homeework18Test.java

```diff
@@ -0,0 +1,55 @@
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+import java.time.Duration;
+
+public class Homeework18Test extends BaseTest {
+
+    @Test
+
+    public void nextSongButton() throws InterruptedException {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--remote-allow-origins=*");
+        options.addArguments("--disable-notifications");
+
+        WebDriver driver = new ChromeDriver(options);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+
+
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
+        emailInput.sendKeys("marianna.tysovska@gmail.com");
+
+        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        passwordInput.sendKeys("marysia288940.");
+
+        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitLogin.click();
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._